### DISCUSSION TITLE
Add speech to text (STT) functionality

### DISCUSSION THREAD

**Discussion by Gottano** [Zain-ul-din/whatsapp-ai-bot#38] [Ideas]
Hi Zain. 

I was thinking it would be great to have this.  Something like OpenAI's Whisper api https://help.openai.com/en/articles/8555505-tts-api

Is this something that I could do in the meantime by editing a particular file? 

Thanks

**Comment by Zain-ul-din** [comment]
@Gottano Thank you for showing your interest in collaboration.

Check these files to get started:

- [Entry Point](https://github.com/Zain-ul-din/whatsapp-ai-bot/blob/master/src/lib/WhatsAppClient.ts)
- [AI Models](https://github.com/Zain-ul-din/whatsapp-ai-bot/tree/master/src/models)
- [GPT Model](https://github.com/Zain-ul-din/whatsapp-ai-bot/blob/master/src/models/ChatGptModel.ts)
